argclass.py

```diff
@@ -629,17 +640,25 @@ def _fill_subparsers(
                     ),
                 )
             )
-            current_parser.set_defaults(current_subparser=subparser)
+            subparser.__parent__ = self
+            current_parser.set_defaults(
+                current_subparsers=subparser._get_chain(),
```

Comment:
why we canot use generator wrapped with tuple?

Reply:
@hugiron for history

@mosquito: When there will be at least 8 depth there, I fucked such a CLI

Reply:
Making AbstractParser an iterator that returns a stack of parents is a strange design. And to do one more abstraction laziness.

Reply:
Iterator protocol was for example) it can be usual method, but ok, let's Keep it our secret))